index.js

```diff
@@ -136,6 +140,36 @@ class Jarvis {
       }
     }
 
+    if (line.startsWith("in this context")) {
+      this.activeConstants = [];
+      const out = 'You are now entering constants. Type the constants, one line at a time. When done, type \'end\'.'
+      return out;
+    }
+
+    if (this.activeConstants) {
+      if (line === 'end') {
+        let keyList = '';
+        this.activeConstants.forEach(constant => {
+          let key = constant.key.toUpperCase();
+          let value = constant.value;
+
+          this.constants[key] = value;
+
+          keyList = keyList != '' ? keyList + ', ' + key : key;
```

Comment:
what's happening here?

Reply:
This is just to make a string to output. I'll update this.